utils/maven-plugin/src/main/java/io/apicurio/registry/maven/JsonSchemaDirectoryParser.java

```diff
@@ -38,15 +38,21 @@ public JsonSchemaDirectoryParser(RegistryClient client) {
     }
 
     @Override
-    public ParsedDirectoryWrapper<Schema> parse(File rootSchemaFile) {
+    public ParsedDirectoryWrapper<SomeJsonSchema> parse(File rootSchemaFile) {
```

Comment:
Have you tested the existing examples with this change? I'm not sure if those are part of the ones executed automatically, so we should check them.

Reply:
Will check. Yeah, we should run examples as tests.